ydb/library/yql/public/udf/arrow/block_builder.h

```diff
@@ -1091,6 +1115,25 @@ std::unique_ptr<TArrayBuilderBase> MakeArrayBuilderBase(
     const ITypeInfoHelper& typeInfoHelper, const TType* type, arrow::MemoryPool& pool, 
     size_t maxBlockLength, const IPgBuilder* pgBuilder);
 
+template<bool Nullable>
+std::unique_ptr<TArrayBuilderBase> MakeResourceArrayBuilder(
+    const ITypeInfoHelper& typeInfoHelper, const TType* type, arrow::MemoryPool& pool, 
+    size_t maxLen) {
+
+    using TBuilder = TFixedSizeArrayBuilder<TUnboxedValue, Nullable, 
```

Comment:
please use distinct class TResourceArrayBuilder - it has too many differences to POD builder